artemis/config.py

```diff
@@ -21,6 +23,43 @@ def get_config(name: str, **kwargs) -> Any:  # type: ignore
     )
 
 
+class SeverityThreshold(enum.Enum):
```

Comment:
what do you think about a separate folder, module_configurations, so that config.py doesn't grow too much?
that way this one could reside e.g. in artemis/modules_config/nuclei.py